packages/admin-ui/src/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { makeDecoratable, withStaticProps } from "~/utils";
+import { DrawerContent } from "./components/DrawerContent";
+import { DrawerFooter } from "./components/DrawerFooter";
+import { DrawerHeader } from "./components/DrawerHeader";
+import { DrawerOverlay } from "./components/DrawerOverlay";
+import { DrawerPortal } from "./components/DrawerPortal";
+import { DrawerRoot } from "./components/DrawerRoot";
+import { DrawerTrigger } from "./components/DrawerTrigger";
+import { DrawerBody } from "~/Drawer/components/DrawerBody";
+import { Icon } from "./components/Icon";
+import { ConfirmButton } from "./components/ConfirmButton";
+import { CancelButton } from "./components/CancelButton";
+
+export interface DrawerProps
```

Comment:
Shall we export this at the end of the file? It's more of a cosmetic change rather than a functional one.

Reply:
It sounds OK, although I've been doing this across all components so far, so I'm thinking not to do anything at this point.

Maybe an ESLint rule or something could help make these things more consistent.